cohort-engine-api-web/src/test/resources/cql/dependency_test-1.0.0/dependency_test-1.0.0.cql

```diff
@@ -0,0 +1,13 @@
+library "Dependency Test" version '1.0.0'
```

Comment:
This library name doesn't match up casing-wise with the filename. Is the discrepancy intentional? Looks like you might be testing an edge case, which would certainly be a good thing, but it caught my eye as incorrect.

Reply:
No, since I'm mocking returns it didn't matter but let me change it for consistency's sake